package.json

```
@@ -50,21 +50,19 @@
     "node": ">=6.0.0"
   },
   "peerDependencies": {
-    "@angular/common": "^7.0.0",
-    "@angular/core": "^7.0.0",
+    "@angular/common": "~8.0.3",
+    "@angular/core": "~8.0.3",
     "rxjs": "^6.0.0",
     "tslib": "^1.9.0"
   },
   "devDependencies": {
-    "@angular/animations": "^7.1.3",
-    "@angular/common": "^7.1.3",
-    "@angular/compiler": "^7.1.3",
-    "@angular/compiler-cli": "^7.1.3",
```

Please add `@angular/compiler-cli` back to devDependencies
```suggestion
    "@angular/compiler-cli": "^7.1.3",
```